python/pylibwholegraph/pyproject.toml

```diff
@@ -27,15 +27,11 @@ authors = [
     { name = "NVIDIA Corporation" },
 ]
 license = { text = "Apache 2.0" }
-requires-python = ">=3.6"
+requires-python = ">=3.10"
```

Comment:
@alexbarghi-nv was it intentional that `pylibwholegraph` has a floor of Python 3.6? It's not being tested against such an older version, but I'm wondering if that was intentionally kept here to support some alternative builds or something.

Reply:
I think there is no specific reason for this. Maybe this is because that this hasn't been updated for a long time. Wholegraph works with python3.10 well. So I think we can update the python version as other rapidsai projects.